Assets/Scripts/GameController.cs

```diff
@@ -330,7 +371,7 @@ public class GameController : MonoBehaviour {
 
 	// Runs every frame
 	void Update() {
-		// Testing move/zoom
+		// Computer move/zoom
```

Comment:
If the updated move/zoom that works on phones now also works for click/drag on computers, this should probably be removed.